sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomModelInfo.cs

```diff
@@ -24,14 +23,14 @@ internal CustomModelInfo(ModelInfo_internal modelInfo)
 
         /// <summary>
         /// </summary>
-        public ModelStatus Status { get; }
+        public CustomFormModelStatus Status { get; }
 
         /// <summary>
         /// </summary>
-        public DateTimeOffset? CreatedOn { get; }
+        public DateTime CreatedOn { get; }
```

Comment:
curious, why the change to `DateTime`?

Reply:
This comment from Paul in the APIView (Rev 40 on CustomModelInfo):

> bojunehsu1 week ago
> We always return UTC time (documented). So I wonder if we can just use DateTime with Kind=Utc.

We'll set the Kind to UTC.

Reply:
Thanks for the pointer Anne.
I remembered we had a conversation about this in KeyVault and went and check and all of our client libraries use DateTimeOffset instead of DateTime. I didn’t find it in our SDK guidelines, but according to the Framework Design Guidelines:
```
 DO use DateTimeOffset whenever you are referring to an exact point in time, e.g. use it to calculate “now”, transaction times, file change times, logging event times, etc. If the time zone is not known, use it with UTC. These usages are much more common than the scenarios where DateTime is preferred, so this should be considered the default.
 DO use DateTime for any cases where the absolute point in time does not apply: e.g. store opening times that apply across time zones.
 DO use DateTime when the time zone is either not known, or is sometimes not known. This may happen in cases where it comes from a legacy data source. 
 DO NOT use DateTimeKind, if DateTimeOffset can be used instead.
DateTimeKind is an enum stored in DateTime to indicate whether the instance represents UTC, local time, or unspecified time zone. 
 DO use DateTime with a 00:00:00 time component, rather than DateTimeOffset, to represent whole dates, e.g. a date of birth.
 DO use TimeSpan to represent times of day without a date.
```
So according to that, we should go back to DateTimeOffset. What do you think?

Reply:
Excellent call out - thank you for the correction, @maririos !  Would you mind posting this in the APIView and I will correct the design?

Reply:
Sure